src/screens/dashboard/DashboardScreen.tsx

```diff
@@ -28,12 +28,21 @@ export class DashboardScreen extends React.PureComponent<NavigationInjectedProps
   render() {
     const student = this.props.navigation.getParam('student');
 
+    const navigateToCreatePlan = () =>
+      this.props.navigation.navigate('PlanActivity', {
+        student,
+      });
+
     return (
       <View style={styles.container}>
         <FullScreenTemplate padded darkBackground>
           {student && <StudentPlanList student={student} />}
         </FullScreenTemplate>
-        <CreateStudentButton />
+        {student ? (
+          <CreateButton actionsName="create plan" onPress={navigateToCreatePlan} />
```

Comment:
Avoid using whitespaces in such strings. If you don't need to display these, use `-, _ ,` etc. to split words.

Reply:
fixed